src/main/java/org/folio/service/transformationfields/TransformationConfigConstants.java

```diff
@@ -45,7 +45,7 @@
     ELECTRONIC_ACCESS_LINKTEXT_DEFAULT,
     ELECTRONIC_ACCESS_MATERIALS_SPECIFIED_DEFAULT,
     ELECTRONIC_ACCESS_PUBLICNOTE_DEFAULT,
-    ID,
+    HOLDINGS_UUID,
```

Comment:
ID is a common field for all types, why we specified it for holdings?

Reply:
yup that was a left over cleanup, changed it now. Thanks !